assets/app/components/site/sitePreviewLinksTable.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+const branchRow = ({ branch, site }) => (
+  <tr key={branch.name}>
+    <td>{branch.name}</td>
+    <td>
+      <a href={previewURL({ branch, site })}>View</a>
+    </td>
+  </tr>
+)
+
+const previewURL = ({ branch, site }) => {
+  if (branch.name === site.defaultBranch) {
+    return site.viewLink
+  } else {
+    return `/preview/${site.owner}/${site.repository}/${branch.name}/`
```

Comment:
I grabbed it from here: https://github.com/18F/federalist/blob/staging/assets/app/components/site/pagesHeader.js#L20

But it could be pulled out of these components and put in a helper somewhere.